ditto/readers/synergi/read.py

```diff
@@ -228,6 +228,7 @@ def parse(self, model):
         LineID = self.get_data("InstSection", "SectionId")
         # FeederId = self.get_data("InstSection", "FeederId")
         LineLength = self.get_data("InstSection", "SectionLength_MUL")
+        LineHeight = self.get_data("InstSection", "AveHeightAboveGround_MUL")
```

Comment:
Is `AveHeightAboutGround_MUL` correct? It almost seems like it was meant to be `AvgHeightAboveGround_MUL` but I'm not familiar with the synergi fields too well.

Reply:
Looking at the files we have, the attribute does seem to be `AveHeightAboveGround_MUL`